examples/node/README.md

```diff
@@ -55,6 +45,5 @@ TUTORIAL
 You can find a more detailed tutorial in [gRPC Basics: Node.js][]
 
 [homebrew]:http://brew.sh
-[linuxbrew]:https://github.com/Homebrew/linuxbrew#installation
-[gRPC install script]:https://raw.githubusercontent.com/grpc/homebrew-grpc/master/scripts/install
-[gRPC Basics: Node.js]:https://github.com/grpc/grpc/blob/master/examples/node/route_guide/README.md
+[Install gRPC Core]:../../INSTALL
```

Comment:
You don't need to install the Node.js library specifically. As long as you have core on your system, you can do `npm install` with a package that depends on grpc (like this examples package), and everything will be installed properly.